store/keystore.go

```diff
@@ -47,19 +50,35 @@ func (k *Keystore) GetKeyName() string {
 	return k.keyName
 }
 
-func NewKeystore(store Store, name string, static bool) *Keystore {
+func NewKeystore(store Store, name string, sslEngine string, static bool) *Keystore {
```

Comment:
There are no unit tests to test this function with `sslEngine != ""`, as it will require a setup similar to the one in `mendersoftware/openssl` repo.

As i understand it, it is covered in the unit tests on `mendersoftware/openssl` and in the soon-to-be integration test with the softhsm. So I am okey with this.

Reply:
yes!